lib/persistence.js

```diff
@@ -6,6 +6,7 @@
  */
 const path = require('path')
 const async = require('async')
+const byline = require('byline')
```

Comment:
I'm not entirely comfortable to use a ~100 lines package that hasn't been updated in 5 years.. The whole point of this fork of nedb is to move away from unmaintained dependencies (there's still a long way, I need to remove or update `async@0.2.10`..), but I'd rather recode and/or include `byline` and its tests inside nedb.

Reply:
I see, the `byline` library does look unmaintained. I searched for an alternative but couldn't find anything better.

..OK, this PR now includes a fork of `byline`. It's been improved based on lint messages ([6884f23](https://github.com/seald/nedb/pull/5/commits/6884f235dde7add6d3753aef8970f9b6ba38130d)).

I'll plan to move over the tests also. Hmm, but I see they include a couple of big test files, total about 1MB. I'm not sure if they belong in the `nedb` repo?

Reply:
Thanks :) I'd be more comfortable if we had all the tests from byline, they are automatically excluded from the npm package anyway.

Reply:
OK, I will move over the tests from byline.